tools/certify/commands/auth-ca.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef CERTIFY_COMMANDS_AUTH_CA_HPP
+#define CERTIFY_COMMANDS_AUTH_CA_HPP
+
+#include "../command.hpp"
+#include <string>
+
+class AuthCaCommand : public Command {
+public:
+    std::string output;
+    std::string sign_key;
+    std::string sign_cert;
+    std::string subject_key;
+    std::string subject_name;
+    int validity_days;
```

Comment:
Avoid public attributes whenever possible (and they seem to be unnecessary for `Command`).

Reply:
Indeed, I previously parsed the arguments inside `options.cpp` and moved it into the commands now.